v1/proto/service/gribi.proto

```diff
@@ -527,6 +527,17 @@ message FlushResponse {
   // The timestamp at which the Flush operation succeeded and was returned to
   // the client. It is specified in nanoseconds since the Unix epoch.
   int64 timestamp = 1;
+  enum Result {
+    // The device has removed all gRIBI objects in the specified network
+    // instance.
+    OK = 0;
```

Comment:
To discuss, is `OK` a reasonable default here, since this means an implementation that didn't go and correctly set anything here will by default say OK, I wonder whether `UNSET` is a better default?

Reply:
wow, good catch! Thanks! Done!